laceworksdk/api/policies.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+Lacework Policies API wrapper.
+"""
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class PoliciesAPI(object):
+
+    def __init__(self, session):
+        """
+        Initializes the PoliciesAPI object.
+
+        :param session: An instance of the HttpSession class
+
+        :return PoliciesAPI object.
+        """
+
+        super(PoliciesAPI, self).__init__()
+
+        self._session = session
+
+    def create(self,
+               policy_type,
+               query_id,
+               enabled,
+               title,
+               description,
+               remediation,
+               severity,
+               alert_enabled,
+               alert_profile,
+               evaluator_id,
+               limit=None,
+               eval_frequency=None,
+               policy_ui={},
+               org=False):
+        """
+        A method to create a new Lacework Query Language (LQL) policy.
+
+        :param policy_type: A string representing the policy type.
+        :param query_id: A string representing the LQL query ID.
+        :param enabled: A boolean representing whether the policy is enabled.
+        :param title: A string representing the policy title.
+        :param description: A string representing the policy description.
+        :param remediation: A string representing the remediation strategy for the policy.
+        :param severity: A string representing the policy severity.
+            ("info", "low", "medium", "high", "critical")
+        :param alert_enabled: A boolean representing whether alerting is enabled.
+        :param alert_profile: A string representing the alert profile.
+        :param evaluator_id: A string representing the evaluator in which the policy is to be run.
+        :param limit: An integer representing the number of results to return.
+        :param eval_frequency: A string representing the frequency in which to evaluate the policy.
+            ("Hourly", "Daily")
+        :param policy_ui: A dictionary which specifies the policyUi domain and subdomain.
+        :param org: A boolean representing whether the request should be performed
+            at the Organization level
+
+        :return response json
+        """
+
+        logger.info("Creating LQL policy in Lacework...")
+
+        # Build the Policies request URI
+        api_uri = "/api/v2/Policies"
+
+        data = {
+            "policyType": policy_type,
+            "queryId": query_id,
+            "title": title,
+            "enabled": int(bool(enabled)),
+            "description": description,
+            "remediation": remediation,
+            "severity": severity,
+            "alertEnabled": int(bool(alert_enabled)),
+            "alertProfile": alert_profile,
+            "policyUi": policy_ui,
+            "evaluatorId": evaluator_id
+        }
+
+        if limit:
```

Comment:
What if we want to set `limit` to zero here?  In that case `limit` would be falsy and this code skipped.  I would recommend:

`if isinstance(limit, int) and limit>=0:`

Reply:
👍